src/main/scala/com/workday/warp/utils/AnnotationReader.scala

```diff
@@ -142,10 +124,9 @@ object AnnotationReader extends StackTraceFilter {
    * @return number of invocations set by the [[Schedule]] annotation.
    */
   def getScheduleInvocations(testId: String): Int = {
-    getWarpTestMethodAnnotation(classOf[Schedule], testId) match {
-      case None => 1
-      case Some(schedule) => schedule.invocations
-    }
+    this.getWarpTestMethodAnnotation(classOf[Schedule], testId)
+      .map(_.invocations())
+      .getOrElse(1)
```

Comment:
should we make a `DEFAULT_INVOCATIONS`?

Reply:
cool i can do that

Reply:
alright there is an already existing default, `Schedule.INVOCATIONS_DEFAULT`.

there are a number of DEFAULT_* and *_DEFAULT. I'll leave those all unchanged. `Schedule`, and `RoundRobinExecution` are JUnit rules that are likely going to be deleted in an upcoming pr